.github/pull_request_template.md

```diff
@@ -1,17 +1,23 @@
 Description
 -----------
-Describe here the proposed change. Thank you for contributing to mitiq! 🙂
+Describe your pull request (PR) here. Thank you for contributing to Mitiq! 🙂
 
 
-If you have not finished all tasks in the checklist, you can also open a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to let the others know this on-going work and keep this checklist in the PR description.
-
 Checklist
 -----------
-Please make sure you have finished the following tasks before requesting a review of the pull request (PR). For more information, please refer to the [Contributor's Guide](../blob/master/CONTRIBUTING.md):
+Check off the following once complete (or if not applicable). The PR will be reviewed once this
+checklist is complete and all tests are passing.
+
+- [ ] I added unit tests for new code.
+- [ ] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures.
```

Comment:
```suggestion
- [ ] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures. These can be checked with `make check-type` from the root directory of the repository, which uses `mypy`.
```

Reply:
mypy checks whether functions are typed correctly, but not whether they are typed. https://mypy.readthedocs.io/en/stable/getting_started.html#function-signatures-and-dynamic-vs-static-typing

Reply:
I was just clarifying that in the Makefile
``` 
.PHONY: check-types
check-types:
	mypy .
```

Reply:
As this is not checked in CI I think its best to ignore. If/when it's checked in CI, I think it's better to let CI check it than the checklist. We can add it to the tips on passing the validate check though.

Reply:
I would agree that until master passes the type check tests as is, it would be hard for contributors to use this and check their own work. Def should be added once that is fixed.